kathara-rest/src/control/lab_wrapper.py

```diff
@@ -21,12 +21,14 @@ class LabWrapper(Lab):
 
     def __init__(self, lab_name: str, lab_state: LabStatus = LabStatus.submitted):
         self.state = lab_state
-        self.path = os.path.abspath(lab_name)
+        lab_dir = os.environ.get('LAB_DIR', os.path.abspath('../labs'))
```

Comment:
The placement of the laboratories is determined by the environment variable `LAB_DIR`. If undefined it defaults to one level above `../labs`.